tests/tck/features/bugfix/SubgraphBeforePipe.feature

```diff
@@ -35,7 +35,7 @@ Feature: Test get subgraph before pipe
       | [:teammate "Tim Duncan"->"Manu Ginobili"@0]     |
       | [:teammate "Tim Duncan"->"Tony Parker"@0]       |
     Then the result should be, in any order, with relax comparison:
-      | _vertices        | _edges    |
+      | VERTICES         | EDGES     |
```

Comment:
lower case is better in database.

Reply:
And our column name is case sensitive.

Reply:
edges. is unreserved_keyword ,  it's not case sensitive